lib/ruby/lib/alces/stack/boot/run.rb

```diff
@@ -100,27 +104,24 @@ def method_missing(s, *a, &b)
 
         def run!
           puts "(CTRL+C TO TERMINATE)"
-          if !@opt.template_parameters.key?("nodename".to_sym) and !@opt.group and !@opt.json 
+          if !@opt.template_parameters.key?(:nodename) and !@opt.group and !@opt.json 
```

Comment:
Prefer `&&` and `||` rather than `and` and `or`. Ref: https://github.com/bbatsov/ruby-style-guide#no-and-or-or